harness.sh

```diff
@@ -283,14 +297,103 @@ tmux send-keys "./ctl createnewaccount c2" C-m
 tmux send-keys "./ctl getnewaddress c2" C-m
 tmux send-keys "./ctl getnewaddress default" C-m
 tmux send-keys "./ctl getbalance"
+
+if [ "${NETWORK}" = "simnet" ]; then
+################################################################################
+# Setup the voting node.
+################################################################################
+cat > "${NODES_ROOT}/vnode/ctl" <<EOF
+#!/bin/sh
+dcrctl -C ../dcrvctl.conf \$*
+EOF
+chmod +x "${NODES_ROOT}/vnode/ctl"
+
+tmux new-window -t $SESSION:4 -n 'vnode'
+tmux send-keys "cd ${NODES_ROOT}/vnode" C-m
+
+echo "Starting ${NETWORK} voting node"
+
+tmux send-keys "dcrd -C ${DCRD_CONF} \
+--rpccert=${DCRD_RPC_CERT} --rpckey=${DCRD_RPC_KEY} \
+--rpcuser=${RPC_USER} --rpcpass=${RPC_PASS} \
+--connect=127.0.0.1:18555 \
+--listen=127.0.0.1:19559 --rpclisten=127.0.0.1:19560 \
+--miningaddr=${CPU_MINING_ADDR} \
+--appdata=${NODES_ROOT}/vnode \
+--txindex \
+--debuglevel=info \
+--simnet" C-m
+
+################################################################################
+# Setup the voting node's dcrctl (vctl).
+################################################################################
+sleep 3
+cat > "${NODES_ROOT}/vnode/mine" <<EOF
+#!/bin/sh
+  NUM=1
+  case \$1 in
+      ''|*[!0-9]*)  ;;
+      *) NUM=\$1 ;;
+  esac
+  for i in \$(seq \$NUM) ; do
+    dcrctl -C ../dcrvctl.conf generate 1
+    sleep 0.5
+  done
+EOF
+chmod +x "${NODES_ROOT}/vnode/mine"
+
+tmux new-window -t $SESSION:5 -n 'vctl'
+tmux send-keys "cd ${NODES_ROOT}/vnode" C-m
+
+# mine some blocks to start the chain if the active network is simnet.
+tmux send-keys "./mine 30" C-m
```

Comment:
This takes 17-18 seconds on my machine. The sleep following this line is only 5 seconds. Does this need to be increased or is this not important?

Reply:
will increase the sleep time